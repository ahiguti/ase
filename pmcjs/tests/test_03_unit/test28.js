
use('Perl');
var testfunc = Perl.Eval("sub { return 'あいうえお'; }");
print(testfunc());
var x = Perl.SuppressConv(function() { return testfunc(); });
dump(x.GetText());
print();
dump(x.GetBinary());
print();
