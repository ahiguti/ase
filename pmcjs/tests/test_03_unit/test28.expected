あいうえお
3042 3044 3046 3048 304a
e3 81 82 e3 81 84 e3 81 86 e3 81 88 e3 81 8a
